app/views/unattended/provisioning_templates/provision/kickstart_default.erb

```diff
@@ -316,6 +316,8 @@ sed -e 's/DEFAULTKERNEL=kernel-uek/DEFAULTKERNEL=kernel/g' -i /etc/sysconfig/ker
 
 <%= snippet 'efibootmgr_netboot' %>
 
+<%= snippet 'insights' %>
+
```

Comment:
A better way is to add the empty line in the snippet itself:
```erb
<% if ... -%>
# install
yum ...

<% end -%>
```

This is a pattern that more snippets should follow.